extensions/film/examples/item.json

```diff
@@ -0,0 +1,25 @@
+{
+  "stac_version": "1.0.0",
+  "stac_extensions": ["https://linz.github.io/stac/_STAC_VERSION_/film/schema.json"],
+  "type": "Feature",
+  "id": "72360",
+  "geometry": null,
+  "properties": {
+    "datetime": "1952-04-23T00:00:00.000Z",
+    "platform": "Fixed-wing Aircraft",
+    "instruments": ["EAGLE IV"],
+    "mission": "SURVEY_1",
+    "film:id": "TL109A",
+    "film:sequence_number": 19,
+    "film:physical_condition": "Metadata manually populated, Flight chart not available",
+    "film:physical_size": "18cm x 23cm"
+  },
+  "links": [],
+  "assets": {
+    "image/tiff; application=geotiff; profile=cloud-optimized": {
+      "href": "./72360.tiff",
+      "type": "image/tiff; application=geotiff; profile=cloud-optimized",
+      "file:checksum": "1220b7deb18ad9dc6f3f94df60c26dd235a019946b8b6b7d1a36f100a8f9f1889130"
+    }
+  }
+}
```

Comment:
Would you be interested in creating any examples of [non-conformant files](https://github.com/linz/stac/pull/56/files)? They are a good way of verifying that the schema actually does what it says.

Reply:
Yep we can do that. A `no_film_id.json`, a `no_negative_sequence.json`, maybe a `negative_sequence_string.json` ..

Reply:
I've added 4 non-examples, missing field and incorrect type for both required fields